src/components/MessageBox.js

```diff
@@ -0,0 +1,96 @@
+const React = require('react');
+
+const StyleConstants = require('../constants/Style');
+const Icon = require('../components/Icon');
+
+const MessageBox = React.createClass({
+  propTypes: {
+    children: React.PropTypes.node,
+    color: React.PropTypes.string,
+    expandable: React.PropTypes.bool,
```

Comment:
👍 to assuming it's expandable if there are children. 

I'm hesitant to add an isOpen prop, unless we just use it as a value to set the initial state (is that ok practice?) I think using callbacks to setProp values in the parent gets messier - this component is a good use case to manage its own state